### ISSUE TITLE
Jekyll Broken

### ISSUE THREAD

**Issue by daggerdagger307** [daggerdagger307/brendanwilson.ca#15]
Hello @thomasjbradley 

Unfortunately my Jekyll has broken and I'm not sure why. It was working up until I closed down brackets and Github then when I restarted them it doesn't link to mark down page. If you have a moment can you help me out? 

Thanks 

-Brendan

**Comment by thomasjbradley** [comment]
I just downloaded the version from GitHub and ran Jekyll and it seems okay, works for me anyways.

You can delete the "portfolio-piece-template.md" file if you're not using it anymore because it does cause a problem.

**Comment by daggerdagger307** [comment]
It works when you go to my Steam Whistle page? All I see is broken links. 

![screen shot 2015-11-10 at 10 56 49 am](https://cloud.githubusercontent.com/assets/6340279/11067290/dfcbb460-8799-11e5-8d89-6b783e381246.png)

**Comment by thomasjbradley** [comment]
Those two images are showing for me:

![screen shot 2015-11-10 at 11 05 29 am](https://cloud.githubusercontent.com/assets/105192/11067542/fbc86cfc-879a-11e5-8ec2-7a2b9be0415d.png)

**Comment by daggerdagger307** [comment]
Wow! weird Thanks I guess it is working. Makes it hard to style it when I can't see it hahaha. I'll figure something out. 

Thanks Thomas

**Comment by thomasjbradley** [comment]
Are you adding the `baseurl` flag when starting Jekyll; this part here: `--baseurl=""`?

**Comment by daggerdagger307** [comment]
Sorry do you mean in my Terminal Window or in my code?

**Comment by daggerdagger307** [comment]
Ya! in my terminal I am

**Comment by thomasjbradley** [comment]
Okay... when you inspect those images in the Web Inspector — What's the URL? What's the `src` show?

**Comment by daggerdagger307** [comment]
Its says       src="/images/"       then on hovering over that it says      http://localhost:4000/images/

**Comment by thomasjbradley** [comment]
Okay, so, the actual image name isn't showing but the path looks right... one moment.

**Comment by thomasjbradley** [comment]
I don't see anything wrong with your code. Is the version on GitHub the most up-to-date one? Have you made unsynced changes?

**Comment by daggerdagger307** [comment]
Yep, I got the newest version of github and I have been making lots of commits. I can just fill out the rest of my mark down file and link them to there pages. I don't know if your off this week but if you have a moment I can come into school and perhaps you can trouble shoot it in person?

**Comment by thomasjbradley** [comment]
Yep, check your email for Wed-Thu times. Plus today is the regular class schedule.

**Comment by daggerdagger307** [comment]
Great thanks Thomas!
